pkg/types/validation/installconfig.go

```diff
@@ -522,22 +522,28 @@ func validatePlatform(platform *types.Platform, fldPath *field.Path, network *ty
 func validateProxy(p *types.Proxy, c *types.InstallConfig, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
 
-	if p.HTTPProxy == "" && p.HTTPSProxy == "" {
-		allErrs = append(allErrs, field.Required(fldPath, "must include httpProxy or httpsProxy"))
+	// Append to error, when there is no proxy configuration set.
+	// If noProxy is set to wildcard char "*", it suggests the configuration is for a transparent proxy
+	if !isInstallConfigHTTPProxySet(p) &&
+		!isInstallConfigHTTPSProxySet(p) {
+		if !isOnlyWildCardSetForNoProxy(p) {
```

Comment:
nit, there's no `else` case here, so you might as well use a single `if` block:

```go
if !isInstallConfigHTTPProxySet(p) &&
  !isInstallConfigHTTPSProxySet(p) &&
  !isOnlyWildCardSetForNoProxy(p) {
```

Or, if you drop the wrappers:

```go
if len(p.HTTPProxy) == 0 &&
  len(p.HTTPSProxy) == 0 &&
  p.NoProxy != "*" {
```